Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -44,7 +44,7 @@ CISOFileReader* CISOFileReader::Create(const std::string& filename)
 
 u64 CISOFileReader::GetDataSize() const
 {
-	return GetRawSize();
+	return CISO_MAP_SIZE * m_block_size;
```

Comment:
I kind of just want it clear to someone glancing at this method that it is returning something that should be constant for all disks.